packages/ui-client/src/hooks/useFeaturePreviewList.ts

```diff
@@ -82,22 +87,46 @@ export const defaultFeaturesPreview: FeaturePreviewProps[] = [
 
 export const enabledDefaultFeatures = defaultFeaturesPreview.filter((feature) => feature.enabled);
 
-export const useFeaturePreviewList = () => {
+export const usePreferenceFeaturePreviewList = () => {
 	const featurePreviewEnabled = useSetting<boolean>('Accounts_AllowFeaturePreview');
-	const userFeaturesPreview = useUserPreference<FeaturePreviewProps[]>('featuresPreview');
+	const userFeaturesPreviewPreference = useUserPreference<FeaturePreviewProps[]>('featuresPreview');
+	// TODO: Remove this logic after we have a way to store object settings.
+	const userFeaturesPreview = useMemo<FeaturePreviewProps[]>(() => {
+		if (typeof userFeaturesPreviewPreference === 'string') {
+			return JSON.parse(userFeaturesPreviewPreference);
+		}
+		return userFeaturesPreviewPreference;
+	}, [userFeaturesPreviewPreference]);
+	const { unseenFeatures, features } = useFeaturePreviewList(userFeaturesPreview);
 
 	if (!featurePreviewEnabled) {
 		return { unseenFeatures: 0, features: [] as FeaturePreviewProps[], featurePreviewEnabled };
 	}
+	return { unseenFeatures, features, featurePreviewEnabled };
+};
+
+export const useDefaultSettingFeaturePreviewList = () => {
+	const featurePreviewSettingJSON = useSetting<string>('Accounts_Default_User_Preferences_featuresPreview');
+	// TODO: Remove this logic after we have a way to store object settings.
```

Comment:
If this is the limitation i wonder if it would be better to extend `multiselect` or create a new setting type that supports this rather than manually doing the parsing when the setting is called.